### PR TITLE
Add Akaunting icon

### LINKED ISSUE #7659: Request: Akaunting
### Brand Name

Akaunting

### Website

https://akaunting.com

### Popularity Metric

The Similarweb rank is 141,388. See https://www.similarweb.com/website/akaunting.com/#overview

### Official Resources for Icon and Color

Url for logo
https://akaunting.com/logo
color: `#6DA252 `

### Additional Comments

_No response_

### Contributing

Yes

### PR DESCRIPTION
**Pull request by Barannaslann**
akaunting

**Issue:** closes #7659

**Similarweb rank:** 141,388
  

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### CODE REVIEW

File: icons/akaunting.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Akaunting</title><path d="M12,19.3c-1.7,0-3.3-0.6-4.5-1.6l-3,3.5c2.1,1.7,4.7,2.7,7.5,2.7c1.9,0,3.7-0.4,5.3-1.2l-2.2-4.1   C14.1,19,13.1,19.3,12,19.3zM4.6,12.3H0.1c0.1,3.4,1.6,6.5,4,8.6L7,17.4C5.6,16.1,4.7,14.3,4.6,12.3zM23.9,12C23.8,5.3,18.6,0.1,12,0.1C5.5,0.1,0.2,5.2,0.1,11.7h4.6c0.2-3.9,3.4-7,7.3-7c4.1,0,7.3,3.3,7.3,7.3   c0,2.7-1.5,5.1-3.7,6.4l2.2,4.1c0.5-0.3,1-0.6,1.5-1v2.4h4.6L23.9,12L23.9,12z"/></svg>
```
Comment (PratyushJayachandran) [inline icons/akaunting.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Akaunting</title><path d="M7.047 17.482a7.311 7.311 0 0 1-2.374-5.131H.078a11.894 11.894 0 0 0 3.976 8.618l2.993-3.487zm8.078 1.226c-.978.461-2.046.7-3.127.697a7.3 7.3 0 0 1-4.51-1.55l-2.992 3.488A11.874 11.874 0 0 0 11.998 24a11.88 11.88 0 0 0 5.292-1.236l-2.165-4.056zm8.796-6.632V24h-4.593v-2.526c-.482.378-.99.722-1.528 1.022l-2.162-4.054a7.329 7.329 0 1 0-10.963-6.668H.076C.159 5.19 5.564-.082 12.15.001s11.856 5.488 11.773 12.074l-.002.001z"/></svg>
```

**Review by PratyushJayachandran — CHANGES_REQUESTED** [review]
Please merge the changes to fix build failure

**Review by adamrusted — CHANGES_REQUESTED** [review]
I'm spotting a slight difference between your PR (black) and the source file (red). This could just be a precision issue, so it'd be worth trying with a precision of 4 instead of 3.

image

Would you mind taking a look at that, @Barannaslann?

**Review by PratyushJayachandran — APPROVED** [review]
Thank you for your contribution @Barannaslann 
Thank you for the correction @adamrusted 
LGTM 🎉

**Pull request merged**